internal/pkg/manifest/validate_env.go

```diff
@@ -68,6 +68,9 @@ func (cfg environmentVPCConfig) validate() error {
 	if err := cfg.Subnets.validate(); err != nil {
 		return fmt.Errorf(`validate "subnets": %w`, err)
 	}
+	if err := cfg.SecurityGroupConfig.validate(); err != nil {
+		return fmt.Errorf(`validate "security group rules": %w`, err)
```

Comment:
I think the error wrapping format we are trying to follow in `Validate` is: `validate "field_name": error`. Applied here it'd be `validate "security_group": %w`.